tests/common.rs

```diff
@@ -1,7 +1,5 @@
 use std::path::PathBuf;
 
-use crate as bitcask;
```

Comment:
because we're moving this module into the integration test, we can just refer to 'bitcask' just as any other external crate

Reply:
this technically broke the benchmarks, I'll fix that in a later commit. benchmarks only run on nightly so there will need to be some changes so its 'valid' on stable.

the [criterion crate](https://docs.rs/criterion/0.3.0/criterion/) does allow for benchmarks on stable, and its generally way more feature-complete than the cargo bench harnass